elements/pf-accordion/acc-lightdom.css

```diff
@@ -0,0 +1 @@
+/* hi */
```

Comment:
`/* hello what are you doing here? */`

Does this file need removed?

Reply:
Thanks for catching that! this was left over from local testing